src/wsiToDcm.cpp

```diff
@@ -277,12 +283,26 @@ int WsiToDcm::wsi2dcm(std::string inputFile, std::string outputFileMask,
         startOnLevel, stopOnLevel, imageName, studyId, seriesId, jsonFile,
         retileLevels,
         std::vector<double>(downsamples, downsamples + retileLevels + 1), tiled,
-        batchLimit, threads);
+        batchLimit, threads, openslideShift);
   } catch (int exception) {
     return 1;
   }
 }
 
+int WsiToDcm::wsi2dcm(std::string inputFile, std::string outputFileMask,
```

Comment:
Can we take this opportunity to clean up this super long parameter list? Instead of having various overloads that are hard to follow, can we use some kind of request struct instead?

Reply:
done